core/src/test/scala/org/apache/spark/sql/delta/ConvertToDeltaSuiteBase.scala

```diff
@@ -98,6 +101,44 @@ trait ConvertToDeltaSuiteBase extends ConvertToDeltaSuiteBaseCommons
     }
   }
 
+  test("convert with collectStats true") {
```

Comment:
so these tests are fine; but now that the default is `true` (and you've updated the test suites accordingly to reflect that) we need tests for when it is explicitly `false`.

Reply:
I wonder how to achieve that in our `ConvertToDeltaScalaSuite`.

Both `ConvertToDeltaScalaSuite` and `ConvertToDeltaSQLSuite` inherite from `ConvertToDeltaSuiteBase`, but since we added this API to the SQL function but still not to the scala `DeltaTable` (its in the second PR), how can we do that?

`HiveConvertToDeltaSuiteBase` does tests collectStats as false